test/new-e2e/tests/containers/dump_cluster_state.go

```diff
@@ -127,7 +127,7 @@ func dumpKindClusterState(ctx context.Context, name string) (ret string) {
 		return
 	}
 
-	if len(instancesDescription.Reservations) != 1 && len(instancesDescription.Reservations[0].Instances) != 1 {
+	if instancesDescription == nil || (len(instancesDescription.Reservations) != 1 && len(instancesDescription.Reservations[0].Instances) != 1) {
```

Comment:
should we check that `len(instancesDescription.Reservations) >= 1` instead?

Reply:
I guess you want also a message when the instance is not found, but no strong opinion.

Reply:
Yes Can be good